src/main/java/com/epam/tc/hw7/pages/MetalsPage.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.tc.hw7.pages;
+
+import com.epam.jdi.light.elements.common.Label;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+import java.util.List;
+
+public class MetalsPage extends WebPage {
+    @UI("#colors .dropdown-menu li span") public static WebList colorsDrop;
+    @UI("#metals .dropdown-menu li span") public static WebList metalsDrop;
```

Comment:
JDI has many special classes for all sorts of elements like checklist, radio button, slider, and so on. This is the essence of this framework.
Ex. Here you can use the class Dropdown.  Then, to select a metal, you can do just  this `metalsDrop.select(metalsColors.metal);`